tmva/tmva/inc/TMVA/ChunkLoader.hxx

```diff
@@ -0,0 +1,108 @@
+#ifndef TMVA_CHUNKLOADER
+#define TMVA_CHUNKLOADER
+
+#include <iostream>
+#include <vector>
+
+#include "TMVA/RTensor.hxx"
+#include "ROOT/RDataFrame.hxx"
+#include "ROOT/RVec.hxx"
+
+namespace TMVA {
+namespace Experimental {
+
+// ChunkLoader class used to load content of a RDataFrame onto a RTensor.
+template <typename First, typename... Rest>
```

Comment:
I see that all the member functions are templated and the `First` and `Rest` arguments of this class template are unused.  Can we fix that, e.g. demoting to non-class-template, or any other fix that is appropriate?